lib/activerecord-import/import.rb

```diff
@@ -313,6 +313,14 @@ def import(*args)
       end
     end
 
+    # Import if all values are valid, otherwise raise an error
```

Comment:
WDYT about mentioning the specific error being raised here and the fact that this will fail-fast? I know the not bang version of this will collect the failed instances and return them all.